be/src/service/backend_service.cpp

```diff
@@ -230,9 +230,19 @@ void BackendService::get_tablet_stat(TTabletStatResult& result) {
 }
 
 void BackendService::submit_routine_load_task(
-        TStatus& t_status, const TRoutineLoadTask& task) {
-    Status status = _exec_env->routine_load_task_executor()->submit_task(task);
-    status.to_thrift(&t_status);
+        TStatus& t_status, const std::vector<TRoutineLoadTask>& tasks) {
+    
+    for (auto& task : tasks) {
```

Comment:
Some tasks may be successful submitted, and some may be fail.
Is this OK?

Reply:
First, it is very unlikely to submit failed. And just make is simple here. Let FE handle the error now.(may be timeout).
It will be changed later if this is a big deal.